paymentsheet/src/test/java/com/stripe/android/paymentsheet/IntentConfirmationHandlerTest.kt

```diff
@@ -875,16 +1103,62 @@ class IntentConfirmationHandlerTest {
         }
     }
 
+    private fun IntentConfirmationHandler.registerAndRetrieveBacsMandateResultCallback():
```

Comment:
Have you considered fakes for these?

Reply:
I think I can add fakes for these with register functions. I'll add them in a quick follow up!